qemu/tests/boot_with_different_vectors.py

```diff
@@ -35,7 +35,7 @@ def boot_guest_with_vectors(vectors):
             env_process.preprocess(test, params, env)
         except virt_vm.VMError as err:
             if int(vectors) < 0:
-                txt = "Parameter 'vectors' expects uint32_t"
+                txt = "expects uint32_t"
                 if re.findall(txt, str(err)):
```

Comment:
```suggestion
                if re.findall(txt, err.output):
```

Reply:
@PaulYuuu Please help me review it again, I have made an update according to the suggested change. Thanks in advance.